### DISCUSSION TITLE
Feature Request: Add multi-line support to Fvwm Modules

### DISCUSSION THREAD

**Discussion by sarishinohara** [fvwmorg/fvwm3#858] [General]
Title is self explanatory i guess;
Might not be a very important feature perhaps but some users such as myself might find it handy.
Working on "FvwmButtons" and  "FvwmScript" As a start would be appreciated, and possibly other modules overtime.

**Comment by ThomasAdam** [comment]
Hi @sarishinohara 

Thanks -- if you have some examples of how you think this should work, let me know.
